inputgraph/loader.go

```diff
@@ -119,9 +119,13 @@ func (l *loader) handleCopy(ctx context.Context, cmd spec.Command) error {
 	return nil
 }
 
+func hasShellExpr(s string) bool {
```

Comment:
nit: call this `mostLikelyHasShellExpre` :laughing:

Reply:
Yeah, it's something of a best effort.  :slightly_smiling_face: